src/utils/constants.js

```diff
@@ -36,6 +36,10 @@ export const MODEL_SERVER_PROPS =  [
   'id', 'ip-addr', 'mac-addr' ,'server-group' ,'nic-mapping' ,'ilo-ip' ,'ilo-user', 'ilo-password'
 ];
 
+export const REPlACE_SERVER_PROPS =  [
```

Comment:
Typo here (the l in REPlACE should be in uppercase)

Reply:
will update